package.json

```diff
@@ -128,19 +128,19 @@
     "jscodeshift": "^0.13.1",
     "madge": "^5.0.1",
     "prettier": "^2.0.5",
-    "react": "^16.10.2",
-    "react-dom": "^16.10.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "typescript": "^4.7.4",
     "webpack": "^4.44.1",
     "webpack-dev-server": "^4.11.1"
   },
   "peerDependencies": {
     "@babel/runtime": "^7",
     "history": "^4 || ^5",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "react": "^18.2.0",
```

Comment:
should that be `"react": "^16.8.0 || ^18.2.0"` to be compatible with react16 in Jira?

Reply:
Done, added types as well